docker-compose.test.yml

```diff
@@ -1,7 +1,13 @@
 version: '2.1'
 services:
+  mysql-ready:
+    image: mysql:5.6
+    depends_on:
+      - mysql
+    command: sh -c "while ! mysqladmin --host=mysql --user=root --password=kuma ping; do sleep 1; done"
```

Comment:
this will run until it reaches the end of the internet, which may be ok if a dev is waiting for the build. 

there is also: https://github.com/pmclanahan/urlwait

Reply:
Oops, good catch @metadave! Thanks for the reference to @pmclanahan 's package. I wasn't aware of that.

Reply:
It is not well advertised :) Feel free to ping me with questions if you decide to go that way.